### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by alexandremoore** [MrFrankel/ngx-popper#38]
In popper-content.ts, it would probably be more appropriate to set the role to `role="tooltip"` instead of `role="popper"`.

The aria-hidden attribute should also be use on the container. `aria-hidden="true"` when the tooltip is not visible and `aria-hidden="false"` when it's showing. Similarly to what is already done with `display: none` vs `display: block`.

Also, it would be great to be able to set an ID so that we could use the aria-describedby attribute on the element where we put the tooltip. 

```
<div [popper]="'As text'"
          [popperTrigger]="'hover'"
          [popperId]="'tooltip1'"
          aria-describedby="tooltip1">
          I have a tooltip
</div>
```

This would generate something like:
```
<popper-content _nghost-c1="">
<div _ngcontent-c1="" id="tooltip1" role="tooltip" aria-hidden="false" class="ngxp__container ngxp__animation" style="display: block; opacity: 1; position: absolute; transform: translate3d(308px, 49px, 0px); top: 0px; left: 0px; will-change: transform;" x-placement="top">        
      <div _ngcontent-c1="" class="ngxp__inner">
        I have a tooltip
      </div>
      <div _ngcontent-c1="" class="ngxp__arrow" style="left: 64px;"></div>  
</div>
</popper-content>
```

Of course, we could workaround this and use our own template and set all required aria attributes there but we can't if we're using simple tooltip with just plain text.

**Comment by MrFrankel** [comment]
@alexandremoore all very good points, as for the aria-describedby, if you use popper-content component you can simply add it to that.
If you use the text option for popper,I wouldn't wand to use the one from the directive as it may not be what the user intends, I guess you can set it as: 
`[popperAriaDescribeBy]="'tooltip1'"`

would that do?

**Comment by vinayakpatil** [comment]
+1

**Comment by MrFrankel** [comment]
in version 2.4.0 i've added aria-hidden property and inputs for
popperAriaRole and popperAriaDescribeBy

**Comment by alexandremoore** [comment]
:+1:

**Comment by alexandremoore** [comment]
@MrFrankel, shouldn't the ariaHidden be set to 'true' as default? Or to !popperShowOnStart?

**Comment by alexandremoore** [comment]
Also, role="popper" is not a valid [ARIA role](https://www.w3.org/TR/html-aria/#allowed-aria-roles-states-and-properties). It should be "tooltip".

**Comment by MrFrankel** [comment]
@alexandremoore ariaHidden is set to the current state of the popper, wether its show or no.

As for role="popper" as default, I will have to leave it for now as its a breaking change and I cant afford the hate.

You can override the default role prop in the global option to tooltip if it is needed

**Comment by alexandremoore** [comment]
@MrFrankel, You can check here: https://stackblitz.com/edit/ngx-popper-issue38 
ariaHidden is false when the page load(it should be true), if you hover the heading the tooltip appears and ariaHidden will still be set to false (as expected). When you mouse out, it will correctly set it to true (also as expected). So the initial state is incorrect, unless one use popperShowOnStart=true.

As for the role, I understand your concern and will override the property in the global options.

Thanks

**Comment by MrFrankel** [comment]
Yes, this is indeed a bug, Ill try and fix it ASAP.
Thanks!

**Comment by alexandremoore** [comment]
Thanks @MrFrankel. 
I also noticed that aria-describedby is added even if we don't specify any value. This violates the aria restriction that aria-{something} attributes must have valid values. https://dequeuniversity.com/rules/axe/2.1/aria-valid-attr-value

Let me know if you need separate bugs for these 2.

**Comment by MrFrankel** [comment]
@alexandremoore indeed this is a bug.

Ill try fix it ASAP

**Comment by MrFrankel** [comment]
@alexandremoore Should be fixed in 2.7.1,

Let me know if its good now.

**Comment by alexandremoore** [comment]
@MrFrankel All good now. Thanks.
